graylog2-web-interface/src/util/conditional/isCloud.js

```diff
@@ -0,0 +1,9 @@
+// @flow strict
+
+import AppConfig from '../AppConfig';
+
+function isCloud(): boolean {
```

Comment:
I'm failing to see why we need this at the moment, could you please explain what the reasoning behind it is?

Reply:
It was a fallback for when we didn't have the `AppConfig`. I'll refactor it out.